CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Unreleased
 * Updated find-up dependency to ^4.0.0
 * Updated postcss-value-parser to ^4.0.0
 * Removed Terra prefix from mixin pattern rule.
+* Updated ESLint to v6
```

Comment:
need to add remove rimraf

Reply:
Updated: https://github.com/cerner/stylelint-config-terra/pull/29/commits/69a97f22ebd3f49c95f0349ffb52088873f84186